### ISSUE TITLE
Implement Fixture Cleanup in `generate_fixture` scripts

### ISSUE THREAD

**Issue by kiepczi** [pyani-plus/pyani-plus#114]
As mentioned in PR #110, we need to ensure that before regenerating fixtures, we remove the existing fixtures using `unlink()`, similar to the implementation in `tests/generate_fixtures/generate_target_dnadiff_files.py`.

```python
for file in DELTA_DIR.glob("*.delta"):
    file.unlink()
for file in FILTER_DIR.glob("*.filter"):
    file.unlink()
for file in SHOW_DIFF_DIR.glob("*.rdiff"):
    file.unlink()
for file in SHOW_COORDS_DIR.glob("*.mcoords"):
    file.unlink()
for file in DNADIFF_DIR.glob("*.report"):
    file.unlink()
```

Currently, this practice is not implemented in other `generate_fixtures` scripts. Implementing this will help maintain consistency and avoid potential issues with outdated test sets.

**Comment by kiepczi** [comment]
I'm just about to create a PR for this. I've already implemented changes to the fastANI `generate_fixture`. I will leave it as it is, and we can resolve any conflicts later, depending on which branch gets merged first, I guess.

**Comment by kiepczi** [comment]
Forgot to mention that, I'm working on this on branch `issue_114`.

**Comment by peterjc** [comment]
Closed by #115.
